icu4c/source/i18n/plurrule_impl.h

```diff
@@ -302,6 +310,7 @@ class U_I18N_API FixedDecimal: public IFixedDecimal, public UObject {
     UBool       isNegative;
     UBool       _isNaN;
     UBool       _isInfinite;
+    int32_t     suppressedExponent;
```

Comment:
Discussed offline: we don't need this field.  suppressedExponent is always 0 in FixedDecimal.

Reply:
Done, and also done in Java.